docs/docs/notification-center/react/react-components.md

```diff
@@ -136,7 +139,11 @@ If you only wish to modify some parts of the existing Novu component UI, you can
 
 :::
 
-## Customize the UI language
+When building your custom UI implementation it might be useful to know, how the notification feed model is structured, so you can customize the notification items during rendering.
```

Comment:
```suggestion
When building your custom UI implementation it might be useful to know how the notification feed model is structured, so you can customize the notification items during rendering.
```